### DISCUSSION TITLE
How Can We Issue Donation Receipts Without Adding Person Entries?

### DISCUSSION THREAD

**Discussion by kernny** [sheltermanager/asm3#1495] [Q&A / Help]
Hi,
We've recently obtained tax-deductible status, and our donors are now requesting receipts. We're looking for the most efficient way to process these without needing to create a PERSON entry for each donation or receipt issued.

Currently, we create an account entry for each donation, but there isn't an option to directly send a receipt from these entries. I'm considering a few alternatives and would appreciate your input or suggestions on these or any options I might have overlooked:

1. In the FINANCIAL section, Add RECEIVE A PAYMENT entry and send receipt directly from that entry ( I'm unsure if this method will affect the existing account entries).

2. From the DOCUMENT REPOSITORY - select the receipt template, complete the details manually and send via email. This method doesn’t auto-populate fields but allows for consistent use of the template.

3. Possibly set up an ONLINE FORM for internal use that can be filled out and submitted to automatically send the receipt. We haven’t used the online forms extensively, so I'm uncertain about the reliability of this solution.

Am I on the right track? Any guidance or additional suggestions would be greatly appreciated. Thanks!

**Comment by kernny** [comment]
Sorry, I've just realised #2 won't work as you have to select a PDF to send the email and as we don't have a PDF, this won't be an option.

**Comment by kernny** [comment]
I don't think #3 will work as forms all appear to be one recipient type set up when the form is created.

**Comment by bobintetley** [comment]
Hi, I'm not sure I'm following - you cannot receive a payment without having a person record. Receipts can be generated with the "Receipt/Invoice" button on the payment tab of an animal or person, or via the Financial->Payment Book screen. You select all the payments you want to appear on the receipt.

Unfortunately, the problem at this point is that if you want to send the receipt as a PDF to the person, you have to save it to their media tab (with the save button after the document is generated) and then use the "Email PDF" option. I've put a ticket on our development list to look at an option to email as PDF from within the document editor. This would allow you to generate the PDF  and email it without having to save the receipt to the media tab of a record.
